content/accessibility/links.mdx

```diff
@@ -0,0 +1,65 @@
+
+People who have low or colorblind vision may have trouble identifying links that just use color to distinguish between plain text, this is why keeping the underline styling on links within body text is important for identification.
+
+## How to test links
+- Check a link's function. If clicking a link instead activates a feature on the page, it should be a `<button>`, and vice versa. 
+- Check a link's purpose. If you can't get an idea of where a link will take you based on the link text alone, the link text needs updates. 
+- [Check a link's contrast](https://webaim.org/resources/contrastchecker/). Links being text supersede the widget contrast requirement, which means all links need to pass 4.5:1 against the background color that they are on. 
+- In body text, like a markdown file, check to make sure that all links have underline styling. 
+
+## Guidelines
+
+### For designers
+- Make sure you're using the proper accent color for links in your mocks. Double check the color you're using if you're using it on anything other than a white background. For more context on color, check out our [Color | Interface guidelines (primer.style)](https://primer.style/design/foundations/color).
+- Make sure a link's purpose can be understood on its own without the surrounding context in concise words. 
+- Links should always be underlined if they're in body text. 
+- Links should look like links, not buttons.
+
+<DoDontContainer stacked>
+    <Do>
+        <Caption>Be sure to underline links in paragraphs and sentences</Caption>
+        <img src="https://user-images.githubusercontent.com/40274682/157987879-8762ad9c-c287-4360-b55f-9d85658e874b.png" alt="Appropriate underlined links in a GitHub issues post">
+    </Do>
+    <Dont>
+        <Caption>Underline every link, or forget to underline any links in paragraphs and sentences</Caption>
+        <img src="https://user-images.githubusercontent.com/40274682/157987884-d1ba14ca-9844-44af-b648-5236dc513a16.png" alt="GitHub issues post with no underlined links">
+    </Dont>
+</DoDontContainer>
+
+### For engineers
+- Use the Primer link components:
+	- [Link | Primer ViewComponents](https://primer.style/view-components/components/link)
+	- [Link | Primer React](https://primer.style/react/Link)
+- Don't override the link styling provided in the components
+- Make sure links receive our global focus indicator styling when navigating the page with a keyboard.
+- Avoid adding side effects to link click events. Links should navigate, not affect the page.
+
+## Common mistakes
+- Link underlines only on focus or hover
+- Link underlines on *every link on the page*
+
+## Additional resources
+- [Color to Distinguish Links from Text](https://dequeuniversity.com/class/visual-design/color/distinguish-links-from-text)
+- Accessibility Insights plugin > Assessment > 7. Links
+- [WebAIM: Links and Hypertext - Introduction to Links and Hypertext](https://webaim.org/techniques/hypertext/)
+
```

Comment:
Do we want to add [Developing a focus style for a themable design system](https://adhoc.team/2022/02/08/creating-focus-style-for-themable-design-system/)?

Reply:
I wouldn't add this one here since it's much more than links.

Reply:
ok, sounds good.